src/libs/tmdk/e2e-tests/basic-e2e/basic-e2e-constants.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as path from "path";
+
+export const localResourcesDir = path.join(__dirname, "basic-e2e-resources");
+export const workspaceId = "tmdk-e2e-test-workspace";
+export const region = "us-east-1";
+export const tmdkDirectory = "/tmp/e2e-tmdk-test";
```

Comment:
just curious if this could make the integ test not work on certain platforms (can add to backlog if so)

Reply:
path.join should use the correct delimiter, but this is a good point for the tmp folder I use for tmdk. I'll update that to use path.join as well
https://nodejs.org/api/path.html